pkg/reconciler/pipelinerun/controller.go

```diff
@@ -59,9 +61,10 @@ func NewController(opts *pipeline.Options, clock clock.PassiveClock) func(contex
 		pipelineRunInformer := pipelineruninformer.Get(ctx)
 		resolutionInformer := resolutioninformer.Get(ctx)
 		verificationpolicyInformer := verificationpolicyinformer.Get(ctx)
-		tracerProvider := tracing.New(TracerProviderName)
+		secretinformer := secretinformer.Get(ctx)
```

Comment:
Is it possible to add an e2e test to make sure the secret informorer/lister we added are working properly?

Reply:
@Yongxuanzhang there is a unit test that exercises this through the secrets fake informer, I think that provides good coverage for that - wdyt?

Reply:
It would be good to have a unit test that verifies the case of secret being updated though.

Reply:
I have added a new test case to cover that - `TestHandlerSecretUpdate`